### PR TITLE
fixing build break due to recent DI changes

### PR DESCRIPTION
**Pull request by maumar**

### CODE REVIEW

**Comment by maumar** [comment]
@bricelam

**Comment by smitpatel** [comment]
and I was stuck on this :cry:

**Comment by pranavkm** [comment]
Are you on an old build of DI? If you're having trouble switch to volatile. You shouldn't need to do this change.

**Comment by bricelam** [comment]
Oh wow, our local mirror (at 15957) is significantly behind the ci (at 15983). @victorhurdugaci?

**Comment by victorhurdugaci** [comment]
There's no more local mirror. There's only a cache of the local coherence. If you need a cache, you have to create job that does that.

We changed it so that all the test builds use the same configuration and do the same thing

**Comment by victorhurdugaci** [comment]
I thought you're not using it for anything else other that the CI after syncing with @natemcmaster

**Comment by victorhurdugaci** [comment]
Btw, the cache is now in `D:\feeds\Coherence`

File: src/EntityFramework.Commands/Design/Internal/StartupInvoker.cs [hunk 1]
```diff
@@ -95,7 +95,7 @@ protected virtual IServiceCollection ConfigureHostServices([NotNull] IServiceCol
             => services
 #if DNX451 || DNXCORE50
                 .ImportDnxServices()
-                .AddSingleton<IHostingEnvironment>(new HostingEnvironment { EnvironmentName = _environment })
+                .AddSingleton<IHostingEnvironment>(_ => new HostingEnvironment { EnvironmentName = _environment })
```
Comment (natemcmaster) [inline src/EntityFramework.Commands/Design/Internal/StartupInvoker.cs]: Why do we need the lambdas? Assuming you are reacting to https://github.com/aspnet/DependencyInjection/commit/ff7eb2f2c9bc584205f143f9ca23feee024f0ee6, it doesn't look  like this needs to change.

Comment (maumar) [inline src/EntityFramework.Commands/Design/Internal/StartupInvoker.cs]: our build is/was using old version of DI, will close this and wait for the right version to propagate

**Pull request closed without merging**